lnrpc/peersrpc/peers_server.go

```diff
@@ -23,7 +25,12 @@ const (
 
 var (
 	// macPermissions maps RPC calls to the permissions they require.
-	macPermissions = map[string][]bakery.Op{}
+	macPermissions = map[string][]bakery.Op{
```

Comment:
Is the intent that we use a _new_ macaroon for this sub-server like some of the existing ones? https://github.com/lightningnetwork/lnd/blob/dd7bfa0c4a8ffe726db2f753ca429fba5387acbf/lnrpc/chainrpc/chainnotifier_server.go#L114

Reply:
I think we discussed this [here](https://github.com/lightningnetwork/lnd/pull/5857#discussion_r734543675) and agreed that we don't want to create individual macaroons anymore?